src/translations/components/Map.ts

```diff
@@ -27,5 +28,29 @@ const MapTexts = {
       heading: _('Filter', 'Filter', 'Filter'),
     },
   },
+  externalRealtimeMap: {
+    bottomSheet: {
+      heading: _('Sanntidskart', 'Real-time map', 'Sanntidskart'),
```

Comment:
Hyphen is used here in the English translation, but not in the one below. Should be the same.

Reply:
Fixed 👍